### PR TITLE
Update create-a-phone-system-auto-attendant.md

### PR DESCRIPTION
**Pull request by poiriersimon**
There is a mistake in the Extension property and add a Powershell exemple

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

**Comment by scanum** [comment]
@MikePlumleyMSFT PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

File: Teams/create-a-phone-system-auto-attendant.md [hunk 1]
```diff
@@ -114,7 +114,8 @@ Users you wish to make available for Dial By Extension need to have an extension
 - TelephoneNumber/PhoneNumber
 - OtherTelephone
 
-The required format to enter the extension in the user phone number field is either *+\<phone number>ext=\<extension>* or *+\<phone number>x\<extension>*.
+The required format to enter the extension in the user phone number field is either *+\<phone number>;ext=\<extension>* or *+\<phone number>;x\<extension>*.
+Ex: Set-MsolUser -UserPrincipalName usern@domain.com -Phonenumber "+15555555678;ext=5678"
```
Comment (JohanFreelancer9) [inline Teams/create-a-phone-system-auto-attendant.md:118]:
```suggestion
Example: Set-MsolUser -UserPrincipalName usern@domain.com -Phonenumber "+15555555678;ext=5678".
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
@poiriersimon - please merge minor edits.

**Comment by MikePlumleyMSFT** [comment]
@CLYVR can you review? In my testing, having a semicolon between number and extension didn't work.

File: Teams/create-a-phone-system-auto-attendant.md (hunk 1 above)
Comment (poiriersimon) [reply Teams/create-a-phone-system-auto-attendant.md:118]: The "." at the end shouldn't be there.

**Comment by poiriersimon** [comment]
@MikePlumleyMSFT in my testing they are needed this is why I put this request as it was causing issue in my environment.

**Comment by CLYVR** [comment]
semicolon is required.

**Pull request merged**